pkgs/tools/package-management/nix-doc/default.nix

```diff
@@ -2,13 +2,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "nix-doc";
-  version = "0.6.4";
+  version = "0.6.5";
 
   src = fetchFromGitHub {
     rev = "v${version}";
     owner = "lf-";
     repo = "nix-doc";
-    sha256 = "sha256-yL0oG0NiQ7OdGQ/kZxQbSbNphKapu5HBFNP5E2fVe+Y=";
+    sha256 = "sha256-9cuNzq+CBA2jz0LkZb7lh/WISIlKklfovGBAbSo1Mgk=";
```

Comment:
```suggestion
    hash = "sha256-9cuNzq+CBA2jz0LkZb7lh/WISIlKklfovGBAbSo1Mgk=";
```

Reply:
In my personal opinion this sha256->hash migration being gone about by bothering individual maintainers piecemeal, without any explicit writing down of a real deprecation or migration plan, or indeed auto migrating the entire code base, is the wrong way of going about it. Implementing tooling to either require migration in a pr check or do it automatically for you is something the nixpkgs architecture team should be doing.

Thanks for the feedback, I'm declining to take action on it absent policy.

Reply:
appreciate your sincerity ! I don't mind either way ( re hash migration )

there is this in nixpkgs manual https://nixos.org/manual/nixpkgs/stable/#compiling-rust-applications-with-cargo

> . cargoHash256 is used for traditional Nix SHA-256 hashes, such as the one in the example above. cargoHash should instead be used for [SRI](https://www.w3.org/TR/SRI/) hashes